CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 * Update dependency gradle to v8.2.1 ([#2122](https://github.com/pinterest/ktlint/pull/2122))
 * Update dependency org.codehaus.janino:janino to v3.1.10  ([#2110](https://github.com/pinterest/ktlint/pull/2110))
 * Update dependency com.google.jimfs:jimfs to v1.3.0 ([#2112](https://github.com/pinterest/ktlint/pull/2112))
+* As a part of public API stabilization, configure `binary-compatibility-validator` plugin for compile-time verification of binary compatibility with previous `ktlint` versions ([#2131](https://github.com/pinterest/ktlint/pull/2131))
```

Comment:
Are there any resources on how to write good changelog entries? 😅 Or should I just copy issue title?

Reply:
> Or should I just copy issue title?

It depends on how good your issue title is. But thsi description is clear.